src/app/(proper_react)/redesign/(authenticated)/user/welcome/scanning/page.tsx

```diff
@@ -63,7 +63,7 @@ export default async function UserWelcomeScanning() {
             const scanListFull = await getAllScanResults(profileId);
             // Store full list of results in the DB.
             await setOnerepScanResults(profileId, scan.id, {
-              data: scanListFull[0],
+              data: scanListFull,
```

Comment:
what was the reason for only setting the first item before?

Reply:
I'm actually not sure - I think this was from an early spike originally.